app/services/feeds/import.rb

```diff
@@ -65,18 +66,18 @@ def call
 
     attr_reader :earlier_than, :users_batch_size, :num_fetchers, :num_parsers, :users
 
-    # TODO: jeremyf, the logic around which users is a bit convoluted; now we need to consider Article policies
-    #
-    # using nil here to avoid an unnecessary table count to check presence
-    def set_users_from(users_scope:, earlier_than:)
-      @users = users_scope.where(id: Users::Setting.with_feed.select(:user_id))
-      @earlier_than = earlier_than
+    # @return [ActiveRecord::Relation<User>] you'll likely want to set @users from this, but
+    #         [@jeremyf]'s choosing not to do that as it makes the implementation just a bit
+    #         cleaner.
+    def extract_users_from(users_scope:, earlier_than:)
```

Comment:
This method name originally gave me a little pause for thought, because as far as I understand it, we're narrowing the scope before extracting the users from the `users_scope` passed in. 

It makes me feel like `extract_authorized_users` or something similar would be a bit more obvious about what it's doing. Just a vague thought!

Reply:
Agreed: this sound more like a `filter` than `extract`

Reply:
Agreed: this sounds more like a `filter` than an `extract, but that's a non-blocking comment

Reply:
Oooh, I like `filter_users_from` and will factor accordingly.